addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java

```diff
@@ -121,6 +123,29 @@ public List<Alert> getExampleAlerts() {
         return alerts;
     }
 
+    @Override
+    public PluginPassiveScanner copy()
+            throws InstantiationException,
+                    IllegalAccessException,
+                    IllegalArgumentException,
+                    InvocationTargetException,
+                    NoSuchMethodException,
+                    SecurityException {
+        RetireScanRule scanRule = new RetireScanRule();
+        scanRule.setRepo(this.repo);
+        Configuration conf = this.getConfig();
+        if (conf == null) {
+            throw new IllegalArgumentException(
+                    "Cannot copy "
+                            + this.getClass().getCanonicalName()
+                            + " : "
+                            + this.getName()
+                            + " as the configuration is null");
+        }
+        scanRule.setConfig(conf);
```

Comment:
Once the above condition is removed then this can be combine with the previous line. Just use getConfig directly without the intermediate assignment